src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -180,7 +183,7 @@ private boolean useMetric(MetricRule rule, Metric metric) {
         String dimensionValue = dimension.getValue();
         if (dimensionSelectKeys.contains(dimensionName)) {
           List<String> allowedDimensionValues = rule.awsDimensionSelect.get(dimensionName);
-          if (!allowedDimensionValues.contains(dimensionValue)) {
```

Comment:
I think this needs a separate setting, as there's cases where we'd want to pass the aws_dimension_select directly to AWS (the billing API docs say it needs this for example).